tools/springboot/springboot_pkg.sh

```diff
@@ -245,7 +258,11 @@ $JAR_COMMAND -uf0 $RAW_OUTPUT .  2>&1 | tee -a $DEBUGFILE
 cd $WORKING_DIR
 # Finally add BOOT-INF/lib
 cd $TMP_LIB_DIR
-$JAR_COMMAND -uf0 $RAW_OUTPUT .  2>&1 | tee -a $DEBUGFILE
+if [ "$USE_BUILD_DEPENDENCY_ORDER" == true ]; then
+  $JAR_COMMAND -uf0 $RAW_OUTPUT $BOOT_INF_LIB_JARS  2>&1 | tee -a $DEBUGFILE
```

Comment:
can you put a a "see docs about Dependency Ordering if this statement blows up" style comment here